gitlab-mr-status/src/main/resources/gitlab/plugin.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<go-plugin id="gitlab.fb.status" version="1">
+    <about>
```

Comment:
The other plugins seem to use a template for this, can we be consistent?

https://github.com/gocd-contrib/gocd-build-status-notifier/blob/master/gerrit-cs-status/src/main/resources/plugin.xml

Reply:
updated